src/enzo/ActiveParticleRoutines.C

```diff
@@ -0,0 +1,508 @@
+/***********************************************************************
+/
+/  ROUTINES FOR THE STAR PARTICLE CLASS
+/
+/  written by: John Wise
+/  date:       March, 2009
+/  modified1:  November, 2011 (converting into active particles)
+/
+/  PURPOSE: Instead of restricting star particles to the typical 
+/           particle attributes, this class gives more functionality 
+/           to them.
```

Comment:
Update the documentation about the purpose of this file.

Reply:
Done